velox/dwio/parquet/tests/ParquetTpchTest.cpp

```diff
@@ -123,163 +146,169 @@ class ParquetTpchTest : public testing::Test {
+               "l_linestatus",
+               "l_quantity",
+               "l_extendedprice",
+               "l_extendedprice * (1.0 - l_discount) AS sum_disc_price",
+               "l_extendedprice * (1.0 - l_discount) * (1.0 + l_tax) AS sum_charge",
+               "l_discount"})
           .partialAggregation(
               {0, 1},
-              {"sum(quantity)",
-               "sum(extendedprice)",
+              {"sum(l_quantity)",
+               "sum(l_extendedprice)",
                "sum(sum_disc_price)",
                "sum(sum_charge)",
-               "avg(quantity)",
-               "avg(extendedprice)",
-               "avg(discount)",
+               "avg(l_quantity)",
+               "avg(l_extendedprice)",
+               "avg(l_discount)",
                "count(0)"})
           .planNode();
 
-  auto plan = PlanBuilder(planNodeIdGenerator)
-                  .localPartition({}, {stage1})
-                  .finalAggregation(
-                      {0, 1},
-                      {"sum(a0)",
-                       "sum(a1)",
-                       "sum(a2)",
-                       "sum(a3)",
-                       "avg(a4)",
-                       "avg(a5)",
-                       "avg(a6)",
-                       "count(a7)"},
-                      {DOUBLE(),
-                       DOUBLE(),
-                       DOUBLE(),
-                       DOUBLE(),
-                       DOUBLE(),
-                       DOUBLE(),
-                       DOUBLE(),
-                       BIGINT()})
-                  .orderBy({0, 1}, {kAscNullsLast, kAscNullsLast}, false)
-                  // Additional step for double type result verification
-                  .project(
-                      {"returnflag",
-                       "linestatus",
-                       "round(a0, cast(2 as integer))",
-                       "round(a1, cast(2 as integer))",
-                       "round(a2, cast(2 as integer))",
-                       "round(a3, cast(2 as integer))",
-                       "round(a4, cast(2 as integer))",
-                       "round(a5, cast(2 as integer))",
-                       "round(a6, cast(2 as integer))",
-                       "a7"})
-                  .planNode();
+  auto plan =
+      PlanBuilder(planNodeIdGenerator_)
+          .localPartition({}, {stage1})
+          .finalAggregation(
+              {0, 1},
+              {"sum(a0)",
+               "sum(a1)",
+               "sum(a2)",
+               "sum(a3)",
+               "avg(a4)",
+               "avg(a5)",
+               "avg(a6)",
+               "count(a7)"},
+              {BIGINT(),
+               DOUBLE(),
+               DOUBLE(),
+               DOUBLE(),
+               DOUBLE(),
+               DOUBLE(),
+               DOUBLE(),
+               BIGINT()})
+          .orderBy({0, 1}, {core::kAscNullsLast, core::kAscNullsLast}, false)
+          // Additional step for double type result verification
```

Comment:
@spershin Sergey is adding logic to allow for slight discrepancies when comparing floating point results in https://github.com/facebookincubator/velox/pull/1108/ . This will allow to remove this workaround. CC: @majetideepak